src/manifolds/Rotations.jl

```diff
@@ -344,6 +344,33 @@ function Random.rand!(
     return pX
 end
 
+"""
+    _get_expdhalf(::Rotations, d)
+
+Calculate `exp(d / 2)`.
+"""
+_get_expdhalf(::Rotations, d) = exp(d / 2)
+"""
+    _get_expdhalf(::Rotations{TypeParameter{Tuple{3}}}, d)
+
+Calculate `exp(d / 2)` for the manifold of `Rotations(3)` based on the Rodrigues' rotation
+formula.
+"""
+function _get_expdhalf(::Rotations{TypeParameter{Tuple{3}}}, d)
```

Comment:
is the `d` in the name necessary? And the `_get`?
I would prefer `_exp_half`? or – if I am correct, this is actually expsqrt?

Reply:
I will rename it to `_exp_half`.